### PR TITLE
standardize cap names

### PR DESCRIPTION
**Pull request by peetucket**

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -4,15 +4,11 @@
...
-set :home_directory, "/home/lyberadmin"
-
-#set :branch, 'master'
-
 # Default branch is :master
 ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
 
 # Default deploy_to directory is /var/www/my_app
-set :deploy_to, "#{fetch(:home_directory)}/#{fetch(:application)}"
+set :deploy_to, "/home/lyberadmin/#{fetch(:application)}"
```
Comment (ndushay) [inline config/deploy.rb]: actually make application the literal as well ....

File: config/deploy.rb [hunk 2]
```diff
@@ -4,15 +4,11 @@
 set :application, 'goobi'
 set :repo_url, 'https://github.com/sul-dlss/goobi-robot.git'
 
-set :home_directory, "/home/lyberadmin"
```
Comment (ndushay) [inline config/deploy.rb]: remove "lock" lines above too (commented out)

File: config/deploy.rb (hunk 1 above)
Comment (peetucket) [inline config/deploy.rb]: fixed and force pushed

File: config/deploy.rb [hunk 3]
```diff
@@ -1,19 +1,11 @@
-# config valid only for Capistrano 3.1
-# lock '3.2.1'
-
-set :application, 'goobi'
```
Comment (ndushay) [inline config/deploy.rb]: We seem to be keeping this set in other PRs -- ask @cbeer ???

**Comment by peetucket** [comment]
apparently, application needs to stay --- put it back in

File: config/deploy.rb (hunk 1 above)
Comment (ndushay) [inline config/deploy.rb]: er .. you unfixed this, no?   copy the literal to line 8?

**Pull request merged**